compiler/rustc_lint/messages.ftl

```diff
@@ -496,6 +496,13 @@ lint_requested_level = requested on the command line with `{$level} {$lint_name}
 
 lint_span_use_eq_ctxt = use `.eq_ctxt()` instead of `.ctxt() == .ctxt()`
 
+lint_static_mut_ref = use of mutable static is discouraged
+    .label = use of mutable static
+    .note = use of mutable static is a hard error from 2024 edition
```

Comment:
This says "use of" but the lint only fires on references, does it? PR description and lint text don't seem be agree with each other.